src/ConfigWrapper/WindowsRegistryConfigWrapper.cs

```diff
@@ -172,9 +195,23 @@ public T[] Get<T>(string key, T[] defaultValue, char[] separators, bool errorOnW
             return value.CastAsT<T[]>(defaultValue, errorOnWrongType);
         }
 
+        /// <inheritdoc/>
+        public override T Get<T>(string key, T defaultValue, bool errorOnWrongType)
+        {
+            key = FullKey(key);
+            return base.Get<T>(key, defaultValue, errorOnWrongType);
+        }
+
+        public override T Get<T>(string key)
+        {
+            key = FullKey(key);
```

Comment:
thinking here to add a optional parameter as the other methods isFullyQualified. to identify the Get for a FullKey

Reply:
ok done.